src/sections/Route.enum.ts

```diff
@@ -17,11 +15,9 @@ export enum Route {
 }
 
 export const RouteWithParams = {
-  COLLECTIONS: (collectionId?: string) => `/collections/${collectionId ?? 'root'}`,
-  CREATE_COLLECTION: (ownerCollectionId?: string) =>
-    `/collections/${ownerCollectionId ?? ROOT_COLLECTION_ALIAS}/create`,
-  CREATE_DATASET: (collectionId?: string) =>
-    `/datasets/${collectionId ?? ROOT_COLLECTION_ALIAS}/create`
+  COLLECTIONS: (collectionId: string) => `/collections/${collectionId}`,
```

Comment:
We should keep the `collectionId` optional for this method to create a route to the collections page and that is because the '/collections' route also matches the collection page, which will load the root collection.
Let me know what you think.

```
export const RouteWithParams = {
  COLLECTIONS: (collectionId?: string) =>
    collectionId ? `/collections/${collectionId}` : Route.COLLECTIONS_BASE,
```

Reply:
It's not needed for our current navigation, but I agree it's good to have it in case we need to do that  👍